test/data/schemas/vm.kiwi.templ

```diff
@@ -83,16 +83,23 @@
         {%- if profile != 'common' and multibuild == True %} profiles="{{ profile }}"
         {%- endif -%}
     >
-        {%- for namespace, packages in content['packages'][pkg_type].items() %}
+	{%- if 'overlayfiles' in content.keys() %}
+	{%- for overlay, overlay_content in content['overlayfiles'].items() %}
+    	{%- if 'name' in overlay_content.keys() %}
+        <archive name="{{overlay_content['name']}}.tar.gz"/>
+    	{%- else %}
+	{%- if profile != 'common' %}
+        <archive name="{{profile}}.tar.gz"/>
+	{%- endif %}
```

Comment:
As mention, this should produce duplicate `archive` tags in case there is more than one `overlayfiles` section in a profile. I think it's ok to have this in the test schema, for now anyway. I'll look into adapting the real schema file.